rustic-babel.el

```diff
@@ -126,7 +135,13 @@ execution with rustfmt."
 
            ;; run project
            (let* ((err-buff (get-buffer-create rustic-babel-compilation-buffer-name))
-                  (params (list "cargo" toolchain "run" "--quiet"))
+                  (params (remove nil (list "cargo"
+                                            (unless (or (null toolchain)
+                                                        (= 0 (string-blank-p toolchain))
```

Comment:
Similar comments as above. I wonder if we can refactor it to a single function to simplify it.